Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2018"
 
 [dependencies]
 http = "0.1.17"
-tapyrus = { git = "https://github.com/chaintope/rust-tapyrus", tag = "v0.4.5", features = ["use-serde", "rand"] }
+tapyrus = { git = "https://github.com/chaintope/rust-tapyrus", tag = "v0.4.7", features = ["use-serde", "rand"] }
```

Comment:
Since it's published to [crates.io](https://crates.io/crates/tapyrus), you don't need to set a repository url like other libraries.